llvm/lib/Target/ARM/ARMBaseRegisterInfo.cpp

```diff
@@ -314,9 +314,6 @@ ARMBaseRegisterInfo::getRegPressureLimit(const TargetRegisterClass *RC,
   default:
     return 0;
   case ARM::tGPRRegClassID: {
-    // hasFP ends up calling getMaxCallFrameComputed() which may not be
-    // available when getPressureLimit() is called as part of
-    // ScheduleDAGRRList.
     bool HasFP = MF.getFrameInfo().isMaxCallFrameSizeComputed()
```

Comment:
I don't think I remember any of this from years ago.

But if you remove the comment, shouldn't you also simplify the code to something like and see if anyone hits the assert?
```
assert(MF.getFrameInfo().isMaxCallFrameSizeComputed());
return 5 - TFI->hasFP(MF);
```